backend/lib/cache/index.js

```diff
@@ -21,27 +21,31 @@ class Cache extends Map {
   }
 
   getCloudProfiles () {
-    return this.get('cloudprofiles').list()
+    return this.get('core.gardener.cloud/cloudprofiles').list()
   }
 
   getQuotas () {
-    return this.get('quotas').list()
+    return this.get('core.gardener.cloud/quotas').list()
   }
 
   getSeeds () {
-    return this.get('seeds').list()
+    return this.get('core.gardener.cloud/seeds').list()
   }
 
   getProjects () {
-    return this.get('projects').list()
+    return this.get('core.gardener.cloud/projects').list()
   }
 
   getShoots () {
     return this.get('shoots').list()
   }
 
   getControllerRegistrations () {
-    return this.get('controllerregistrations').list()
+    return this.get('core.gardener.cloud/controllerregistrations').list()
+  }
+
+  getResourceQuotas () {
+    return this.get('core/resourcequotas').list()
```

Comment:
The access methods on the cache do not have an apiGroup prefix. We have no duplicate names. Why do we need the apiGroup internally. Shouldn't we remove it? What do you think?